bitrise.yml

```diff
@@ -2794,6 +2794,187 @@ workflows:
         is_expand: false
       BITRISE_SCHEME: Firefox
     description: ''
+  NewXcodeVersions:
```

Comment:
Note: This is the first time adding a workflow from here, so let's see how this works...if it appears right away in the Bitrise UI to select it to schedule a build or there is something else on Bitrise side. As a workaround I propose to use for example `aaronmt_chrontab_perftests` modified accordingly and rename it in the future when we know creating/updating workflows from bitrise.yml works.